src/univariate/continuous/loguniform.jl

```diff
@@ -0,0 +1,82 @@
+"""
+    LogUniform(a,b)
+
+The *log uniform* distribution also known as *reciprocal distribution* is the exponential
+of the [`Uniform`](@ref) distribution `Uniform(log(a), log(b))`.
```

Comment:
"exponential of the distribution", I think it is not clear what that means. Perhaps follow Wikipedia and define `A positive random variable X is log-uniformly with parameters `a` and `b` if the logarithm of X is Uniform(log(a), log(b)).`